core/bentley/src/BentleyError.ts

```diff
@@ -61,6 +61,7 @@ export enum IModelStatus {
   MissingId = IMODEL_ERROR_BASE + 33,
   NoGeometry = IMODEL_ERROR_BASE + 34,
   NoMultiTxnOperation = IMODEL_ERROR_BASE + 35,
+  // SWB What does project mean here?
```

Comment:
@kabentley @pmconne do we still need this error status `NotDgnMarkupProject`?

Reply:
If not objections, I'd remove this.

Reply:
I don't see any corresponding native status code and a "dgn markup project" was a special DgnDb that held markup info for another read-only DgnDb back in bim02/DgnDb06 days - I think it's a dead concept.

Reply:
I have gone ahead and removed it.